customizations.md

```diff
@@ -101,11 +101,11 @@ If all ECUs do have additional storage, and can perform a rollback, then the OEM
 
 **Figure 2.** *Two options for fleet management with Uptane.*
 
-Some parties, such as vehicle rental companies or the military, may wish to exercise control on how their own fleet of vehicles are updated. Uptane offers two options for implementing fleet management, as illustrated in Figure 2. Choosing between these options depends on whether the fleet manager wishes to have either complete control, or better compromise-resilience.
+Some parties, such as vehicle rental companies or the military, may wish to exercise control on how their own fleet of vehicles are updated. Uptane offers two options for serving these users, as illustrated in Figure 2. Choosing between them depends on whether the fleet manager wishes to have either complete control, or better compromise-resilience.
 
-In the first option, which we expect to be the common case, a fleet manager would configure the map file on ECUs, such that Primaries and full verification Secondaries would trust an image if and only if it has been signed by both the Image repository managed by the OEM, and the Director repository, which would be managed by the fleet. Partial verification Secondaries would trust an image if and only if it has been signed by this Director repository. The upside of this option is that the fleet manager, instead of the OEM, has complete control over which updates are installed on its vehicles. The downside of this option is that, if the Directory repository controlled by the fleet manager is compromised, then attackers can execute mix-and-match attacks.
+In the first option, which we expect to be the common case, a fleet manager would configure the map file on ECUs such that Primaries and full verification Secondaries would trust an image if, and only if, it has been signed by both the OEM-managed Image repository, and the fleet-managed Director repository. Partial verification Secondaries would trust an image, if and only, if it has been signed by this Director repository. The upside of this option is that the fleet manager, instead of the OEM, has complete control over which updates are installed on its vehicles. The downside of this option is that should the fleet-managed Directory repository be compromised, attackers can execute mix-and-match attacks.
```

Comment:
I added those commas on the last go-ahead for emphasis. That is to add some weight to the decision. If the commas are distracting, maybe we should italicize only. I think we want that "and only if" phrase to stand out. See if below is better...

Reply:
I like the rephrasing with the key word "only". But I wonder if it would be better to emphasize the word "both" instead; I think that's now the critical part.

How did you feel about my change to the last sentence? I found the "should ... be" construction just a bit harder to understand than "if ... is". So if I combine all of that, I'd end up with:
```suggestion
In the first option, which we expect to be the common case, a fleet manager would configure the map file on ECUs such that Primaries and full verification Secondaries would only trust an image that has been signed by *both* the OEM-managed Image repository and the fleet-managed Director repository. Partial verification Secondaries would only trust an image if it has been signed by the fleet-managed Director repository. The upside of this option is that the fleet manager, instead of the OEM, has complete control over which updates are installed on its vehicles. The downside of this option is that if the fleet-managed Directory repository is compromised, attackers can execute mix-and-match attacks.
```